README.md

```diff
@@ -86,20 +87,20 @@ functions:
     vpcDiscovery:
       vpcName: '${opt:env}'
       securityGroupNames:
-        - '${opt:env}_NAME OF SECURITY GROUP'
+        - '${opt:env}_<name of security group>'
   example4:
     handler: handler.example
     # override basic subnet names and security group names
     vpcDiscovery:
       vpcName: '${opt:env}'
       subnetNames: # optional if securityGroupNames are specified
-        - '${opt:env}_NAME OF SUBNET'
+        - '${opt:env}_<name of subnet>'
       securityGroupNames:  # optional if subnetNames are specified
-        - '${opt:env}_NAME OF SECURITY GROUP'        
+        - '${opt:env}_<name of security group>'        
 ```
-> NOTE: The naming pattern we used here was building off the vpc name for the subnet and security group by extending it with the the subnet and security group name. This makes it easier to switch to different vpcs by changing the environment variable in the command line
- 
-> NOTE: The core sls `provider.vpc` config will change the plugin behavior.
+> NOTE: The core serverless `provider.vpc` will inherit or extended functions vpc config after the plugin apply.
```

Comment:
```suggestion
> NOTE: The core serverless `provider.vpc` settings will be used, if they are set, instead of `vpcDiscovery`. You can use also mix settings. For example you may set `provider.vpc. subnetIds` while using `vpcDiscovery` to set the `securityGroupIds` 
```